### DISCUSSION TITLE
MQTT Repeater

### DISCUSSION THREAD

**Discussion by DerBo19** [Blackymas/NSPanel_HA_Blueprint#2130] [Q&A]
Hi guys, 

Maybe im getting something wrong but is there any possibility to add a MQTT Repeater as function for the panel? I have a few zigbee lamps and want to control it with Home Assistant. Unfortunately they are out of the range from the sky connect stick. Was wondering if I can use the panel as a repater, like all zigbee devices. I know zigbee and MQTT are different but I'm using zigbee2mqtt.

Thanks for help!

**Comment by grigi** [comment]
The NSPanel doesn't have a Zigbee radio, so no can do.
The NSPanel pro, I think supports zigbee. That's a totally different panel.
